libtsuba/src/RDG.cpp

```diff
@@ -308,7 +309,7 @@ tsuba::RDG::AddPartitionMetadataArray(
         "current graph");
     set_local_to_global_vector(std::move(col));
   } else {
-    return tsuba::ErrorCode::InvalidArgument;
+    return KATANA_ERROR(ErrorCode::InvalidArgument, "chedking metadata name");
```

Comment:
Down with the Brie King, long live the Ched King!

Reply:
All HAIL